apps/resources/models.py

```diff
@@ -115,3 +116,29 @@ def get_most_tried(user=None, limit=1):
             '-most_tried',
         )[:limit]
         return resources
+
+    def get_related(self, user=None, limit=3):
+        data = {}
+        results = {}
+        ids_with_tags = Resource.objects.approved(
```

Comment:
I think you probably need to limit this to resources with matching tags - however that can wait for another day.